packages/template-retail-react-app/jest-setup.js

```diff
@@ -79,13 +79,7 @@ afterEach(() => {
     global.server.resetHandlers()
 })
 afterAll(() => {
-    // Intentionally not closing the server!
-    // We run into many race condition issues,
-    // that was cause by the server close too soon
-    // and the tests not well written in an proper async manner.
-    // Let's not close the server and see how things goes.
-    // We can revisit this.
-    // global.server.close()
+    global.server.close()
```

Comment:
From what I see, bringing it back does not seem to affect anything. But it was the recommended way by msw documentation.